### PR TITLE
Add documentation for Dropwizard

### PR DESCRIPTION
**Pull request by elenacoscale**

### CODE REVIEW

File: _docs_agent/plugins/dropwizard.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+server:
+  requestLog:
+    appenders:
+      - type: file
+        logFormat: '%h "%r" %D %s %b'
+        currentLogFilename: /var/log/dropwizard-app/access.log
+        archivedLogFilenamePattern: /var/log/dropwizard-app/accedd-%d.log.gz
+{% endhighlight %}
+
+Restart Dropwizard aplication to apply these changes.
```
Comment (kidk) [inline _docs_agent/plugins/dropwizard.md]: 'aplication' > 'application'

This is probably also wrong in the app, can you create a ticket for it?

File: _docs_agent/plugins/dropwizard.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    - type: http
+      port: 8081
+{% endhighlight %}
+
+Restart the application to apply these changes.  
+Provide the metrics URL, as you configured it on your server.
+
+## Log configuration
+
+Enable the Dropwizard aplication access log format. Add the following lines to your application YAML configuration file:
```
Comment (kidk) [inline _docs_agent/plugins/dropwizard.md]: 'aplication' > 'application'

Comment (elenacoscale) [reply _docs_agent/plugins/dropwizard.md]: fixed

File: _docs_agent/plugins/dropwizard.md (hunk 1 above)
Comment (elenacoscale) [reply _docs_agent/plugins/dropwizard.md]: fixed

**Comment by kidk** [comment]
Thanks Elena 💯 / 💯

**Pull request merged**